.github/test-a.md

```diff
@@ -0,0 +1 @@
+sdfads
```

Comment:
What's this file used for? If it's not necessary, please remove it.

Reply:
Sorry, this file is added by accident, I've removed it.